src/components/Backdrop/index.js

```diff
@@ -40,6 +46,22 @@ export default class Backdrop extends Component {
     }
   };
 
+  attachListeners = () => {
+    if (document) {
```

Comment:
I think this check will throw on SSR. Please use `process.browser` or `typeof window !== object`

Reply:
OK